src/components/Dialogs/Dialogs.js

```diff
@@ -56,13 +68,13 @@ const Dialogs = (props) => {
 
 Dialogs.defaultProps = {
   sendMessage: PropTypes.func,
-  dialogsPage: PropTypes.func,
+  dialogsPage: PropTypes.instanceOf(Object),
```

Comment:
Consider researching "PropTypes.shape( {...} )" approach and start defining the details of the objects in props.

Reply:
I will think about this a bit later. At the moment, I need to at least fulfill the planned. I fix all types currently